src/Interfaces.tsx

```diff
@@ -2,7 +2,7 @@ export interface UserData {
   token: string,
   user: {
     data: {
-      id: number | string | undefined,
+      id: number,
       type: string,
```

Comment:
I had changed this one because I was getting a typescript flag in MenuBar.tsx; was still hunting the type discrepancy but, small potatoes and easy fix!